src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -37,15 +37,13 @@ final class CurlResponse implements ResponseInterface, StreamableInterface
     private $debugBuffer;
 
     /**
-     * @param \CurlHandle|resource|string $ch
-     *
      * @internal
      */
-    public function __construct(CurlClientState $multi, $ch, array $options = null, LoggerInterface $logger = null, string $method = 'GET', callable $resolveRedirect = null, int $curlVersion = null)
+    public function __construct(CurlClientState $multi, \CurlHandle | string $ch, array $options = null, LoggerInterface $logger = null, string $method = 'GET', callable $resolveRedirect = null, int $curlVersion = null)
```

Comment:
let's ignore fabbot on union types :)

Reply:
All right. We should REALLY get that fixed, though. 😓